test/python/circuit/test_extensions_standard.py

```diff
@@ -851,6 +851,38 @@ def test_z_reg_inv(self):
         self.assertEqual(instruction_set[1].qubits, (self.qr[1],))
         self.assertEqual(instruction_set[2].operation.params, [])
 
+    def test_global_phase(self):
+        qc = self.circuit
+        qc.append(GlobalPhaseGate(0.1), [])
+        self.assertEqual(self.circuit[0].operation.name, "global_phase")
+        self.assertEqual(self.circuit[0].operation.params, [0.1])
+        self.assertEqual(self.circuit[0].qubits, ())
+
+    def test_global_phase_inv(self):
+        instruction_set = self.circuit.append(GlobalPhaseGate(0.1), []).inverse()
+        self.assertEqual(len(instruction_set), 1)
+        self.assertEqual(instruction_set[0].operation.params, [-0.1])
+
+    def test_global_phase_matrix(self):  # , theta: float, expected: np.ndarray
+        """Test global_phase matrix."""
+        theta = 0.1
+        np.testing.assert_allclose(
+            GlobalPhaseGate(theta).__array__(),
+            np.array([[np.exp(1j * theta)]], dtype=complex),
+            atol=1e-7,
+        )
+
+    def test_global_phase_consist(self):
+        """Tests compatibility of GlobalPhaseGate with QuantumCircuit.global_phase"""
+        theta = 0.1
+        qc = QuantumCircuit(0, global_phase=theta)
+        result_qc = Statevector(qc)
+        np.testing.assert_allclose(
+            np.array([[result_qc[0]]]),
+            GlobalPhaseGate(theta).__array__(),
+            atol=1e-7,
+        )
```

Comment:
Please rewrite this test. It should create two circuits, not one. The other circuit should be a circuit with a GlobalPhaseGate. Then, check the consistency by making sure `np.array(Operator(circuit1))` and `np.array(Operator(circuit2))` are the same.

Reply:
Changed the test! Thanks for helping me with that! :)